--- src/main/java/dev/femrek/openapidartnetworkcodegen/DartNetworkClientCodegen.java
+++ src/main/java/dev/femrek/openapidartnetworkcodegen/DartNetworkClientCodegen.java
@@ -194,6 +194,15 @@
             model.imports.clear();
             model.imports.addAll(newImports);
+
+            if (model.name.contains("ApiResponseMapPermissionActionListPermissionTargetType")) {
+                for (CodegenProperty param : model.vars) {
+                    LOGGER.info("Property: " + param.name + ", dataType: " + param.dataType + ", datatypeWithEnum: " + param.datatypeWithEnum + ", isMap: " + param.isMap + ", isArray: " + param.isArray + ", items: " + (param.items != null ? param.items.dataType : "null"));
+                }
+            }
         }
 
         return objs;
